README.md

```diff
@@ -98,5 +98,14 @@ colcon build --symlink-install
 . install/setup.bash
 ```
 
+#### 8. Install and run rosdep
+```branch
+sudo apt install python3-rosdep
+sudo rosdep init
+rosdep update
+rosdep install --from-paths src --ignore-src -r -y
+```
+Make sure to call the 'rosdep install' command from the colcon workspace directory.
```

Comment:
Probably just clarify that the colcon workspace directory is `colcon-urc`, new users really won't know what you mean otherwise